src/py/service/Request.py

```diff
@@ -15,14 +20,91 @@ class Request:
     client_id: int = None
     professional_id: int = None
     service_id: int = None
+    request_status_id: int = None
+    status_name: str = None
 
-    def create_request(self, request_date: datetime, start_date: datetime, completion_date: datetime,
-                       instruction: str, client_id: int, professional_id: int, service_id: int):
-        pass
+    def create_request(self):
+        database = Database.database_handler(DatabaseLookups.User)  # create database connection
 
-    def update_request(self, request_id: int, request_date: datetime, start_date: datetime, completion_date: datetime,
-                       instruction: str, client_id: int, professional_id: int, service_id: int):
-        pass
+        # check if database is connected, if not connect
+        if database.status is DatabaseStatus.Disconnected:
+            database.connect()
+
+        elif database.status is DatabaseStatus.NoImplemented:
+            raise errors.InternalError  # change with mysql errors
+            # attempt to create address row
+        database.select(('request_status_id',), 'request_status')
+        database.where('status_name = %s', self.status_name)
+        self.request_status_id = database.run()[0][0]
+        database.clear()
+
+        # attempt to create address row
+        database.insert(self, 'request', ('request_id', 'professional_id', 'completion_date', 'request_status_id', 'status_name',))
+
+        try:
```

Comment:
Looks like you are trying to create address information, I am not sure why this is?